src/services/pcn-iptables/src/defines.h

```diff
@@ -54,9 +54,10 @@ const uint8_t IPDESTINATION = 2;
 const uint8_t L4PROTO = 3;
 const uint8_t PORTSOURCE = 4;
 const uint8_t PORTDESTINATION = 5;
-const uint8_t TCPFLAGS = 6;
-const uint8_t BITSCAN = 7;
-const uint8_t ACTION = 8;
+const uint8_t INTERFACE = 6;
+const uint8_t TCPFLAGS = 7;
+const uint8_t BITSCAN = 8;
+const uint8_t ACTION = 9;
 }
```

Comment:
I know this is out of scope of this PR. Wouldn't it make sense to use an enum in this case?

Reply:
You are right. `enum` would be a cleaner solution.
I'll keep track of that, and update in a following PR.